Corona-Warn-App/build.gradle

```diff
@@ -32,15 +32,29 @@ def environmentExtractor = { File path ->
 }
 
 android {
+    println("Current VERSION_MAJOR: ${VERSION_MAJOR}")
```

Comment:
we don't really need  the println,  do we

Reply:
Not for us, but I think it's an easy addition which will allow pipeline development to check in the gradle output/logs what values were read. Can remove it in a future PR when the new pipeline is :rocket:.